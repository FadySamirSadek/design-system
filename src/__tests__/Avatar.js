import React from 'react'
import renderer from 'react-test-renderer'
import { Avatar } from '..'

describe('Avatar', () => {
  test('renders', () => {
    const json = renderer
      .create(<Avatar src="https://hackclub.com/team/zach.png" size={64} />)
      .toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('width', '64px')
  })
})
