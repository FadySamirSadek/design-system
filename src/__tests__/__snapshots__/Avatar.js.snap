// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders 1`] = `
.c0 {
  border-radius: 9999px;
  display: inline-block;
  width: 64px;
  height: 64px;
}

<img
  className="c0"
  size={64}
  src="https://hackclub.com/team/zach.png"
/>
`;
